README.md

```diff
@@ -56,7 +58,7 @@ Get in touch with us on [Slack](https://aif360.slack.com) (invitation
 
 ## Setup
 
-AIF360 is available for both Python and R. For R setup, please refer to instructions [here](aif360/aif360-r/README.md).
+### Python
```

Comment:
I would add R first and then Python since the instructions that follow are for Python

Reply:
@gdequeiroz I have removed the line from `setup` and added under first paragraph. Arranged in alphabetical order.

Reply:
I did that because users will know when they start reading through the readme that it is available in both versions

Reply:
I agree with @gdequeiroz, @SSaishruthi can we move R instructions above Python? It's weird to me to have Python, then R, then back to Python instructions.